tests/test_encrypted_data_encryption.py

```diff
@@ -173,14 +173,14 @@ def Main(args=None):
         session_key = utility.generate_key()
         session_iv = utility.generate_iv()
         encrypted_session_key = utility.generate_encrypted_key(session_key,
-                worker_obj.worker_encryption_key)
+                worker_obj.encryption_key)
         # Generate one time key used for encrypting inData
         data_key = utility.generate_key()
         # Initialation vector of size 12 bytes with all zeros.
```

Comment:
s/Initialation/Initialization/

Reply:
This file is out of this commit, will remove it